core/commands/ping.go

```diff
@@ -27,10 +27,8 @@ type PingResult struct {
 
 var PingCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Send echo request packets to IPFS hosts.",
-		Synopsis: `
-Send pings to a peer using the routing system to discover its address
-		`,
+		Tagline:  "Send echo request packets to IPFS hosts.",
+		Synopsis: "Send pings to a peer using the routing system to discover its address.",
```

Comment:
Since youre changing/correcting the text in a bunch of other commands, lets fix this one too. The purpose of ping is to verify connectivity and determine latency, that it uses the routing system to discover addresses is an implementation detail we don't need to mention here.